src/Servers/Kestrel/Core/src/Internal/Http3/QPack/DynamicTable.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3.QPack
+{
+    //   The size of the dynamic table is the sum of the size of its entries.
+    //   The size of an entry is the sum of its name's length in bytes (as
+    //   defined in Section 4.1.2), its value's length in bytes, and 32.
+
+    internal class DynamicTable
```

Comment:
This looks identical to HPACK `DynamicTable`, can we reuse?

Reply:
No, there are differences I believe in the dynamic table (takes different instructions from decoder stream). Eventually, we may be able to combine the tables but I'd rather not bother for now.